src/python/grpcio/grpc/_cython/_cygrpc/aio/call.pyx.pxi

```diff
@@ -19,6 +19,27 @@ _EMPTY_FLAGS = 0
 _EMPTY_METADATA = None
 _OP_ARRAY_LENGTH = 6
 
+class AioCallCancelStatus:
```

Comment:
We can make it a `cdef class`, and make those two values `readonly` from Python layer.

Also, I would prefer its name to be more generic, so the server side can use it as well.